tests/Giraffe.Tests/HttpHandlerTests.fs

```diff
@@ -1187,4 +1187,11 @@ let ``Get "/flex" without an Accept header returns a JSON object`` () =
     | Some ctx ->
         let body = getBody ctx
         Assert.Equal(expected, body)
-        Assert.Equal("application/json", ctx.HttpContext.Response |> getContentType)
\ No newline at end of file
+        Assert.Equal("application/json", ctx.HttpContext.Response |> getContentType)
+
+[<Fact>]
+let ``Warbler function should execute inner function`` () =
+    let inner (a: int) = a.ToString()
+    let warbled = warbler (fun (a:int) -> inner)
+    let result = warbled 42
+    Assert.Equal("42", result)
```

Comment:
Great test, that does indeed cover what we are planning on using the `warbler` for.